academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+
+IBC is an end-to-end, connection-oriented, stateful protocol for reliable, ordered, and authenticated communication between heterogeneous blockchains arranged in an unknown and dynamic topology. It is possible by specifying a set of data structures, abstractions, and semantics that can be implemented by any distributed ledger provided they satisfy a small set of requirements.
+
+<HighlightBox type="info">
+
+More details on the specifications will follow in the next section, but notice that IBC is not limited to Cosmos blockchains. Solutions can even be found for cases where some requirements are not initially met. An example of such a case is the use of IBC for Proof-of-Work (PoW) blockchains like Ethereum. As a PoW consensus algorithm does not ensure finality, one of the main requirements to use IBC is not met. Still, Ethereum-compatibility is enabled by creating a peg-zone where probabilistic finality is considered deterministic (irreversible) after a given threshold of block confirmations.
+
+</HighlightBox>
+
+Although application-specific blockchains offer superior scalability compared to general-purpose blockchain platforms, smart contract development for general-purpose chains and generic virtual machines (VMs) like in Ethereum offer their own benefits. IBC provides a method of incorporating the strengths of general-purpose and application-specific blockchains into unified overall designs. For example, it allows a Cosmos chain tailored towards performance and scalability to use funds that originate on Ethereum and possibly record events in a Corda distributed ledger or, in the reverse, a Corda ledger initiating the transfer of underlying assets defined in Cosmos or Ethereum.
+
+![Internet of blockchains](images/internetofchains.png)
+
+With cross-chain communication via IBC, a decentralized network of independent and interoperable chains exchanging information and assets is possible. This "internet of blockchains" brings the promise of increased and seamless scalability. In Cosmos, the vision being implemented is to have a universe of independent chains that are all connected using peg-zones as bridges between the Cosmos network and chains outside of it and connecting all chains via hubs. All of these make up the internet of blockchains.
+
+## Interchain Standards (ICS)
+
+There are two categories (aside from __meta__) that define ICS in the [general ICS definition](https://github.com/cosmos/ibc/blob/master/spec/ics-001-ics-standard/README.md):
+
+* **IBC/TAO:** a standard defining an inter-blockchain communication system core transport, authentication, and ordering-layer protocol.
+* **IBC/APP:** a standard defining an inter-blockchain communication system application-layer protocol.
```

Comment:
> * **IBC/APP:** a standard defining an inter-blockchain communication system application-layer protocol.

standards defining the application handlers for the data packets being passed over the transport layer. These include but are not limited to fungible token (ICS20) transfers, NFT (ICS721) transfers, and Interchain Accounts (interchain messaging/direction passing between two blockchains). 

include relevant links above when we have them.